apps/responsive-app/src/app/app.component.html

```diff
@@ -43,6 +43,14 @@
           linkActive="mat-list-single-selected-option"
           >Profile 5</a
         >
+        <a
+          mat-list-item
+          [linkTo]="'../6' | valueToAsyncValue | async"
```

Comment:
I think using an Observable variable as a parameter over the `valueToAsyncValue` pipe would be easier to read and closer to what users will actually do in their projects.

Reply:
Ya, I started with that but opted for this due to the `delayTime(10)`. I can inline if if you think that's better.